<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mohits-MacBook-Air.local" name="Default test" tests="55" failures="3" timestamp="11 Jun 2018 03:52:37 GMT" time="4.262" errors="0">
  <testcase name="additionTest" time="1.935" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="divisionTest" time="0.055" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="intersectionTest" time="0.031" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="maxTest" time="0.029" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="minTest" time="0.033" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="multiplytest" time="0.047" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="unionTest" time="0.028" classname="com.automation.tests.APIIntegrationTest"/>
  <testcase name="multiplytest1_Non_Integers_400BadRequest" time="0.044" classname="com.automation.tests.MultiplyServiceTest"/>
  <testcase name="multiplytest2_OneNumberAsNull_400BadRequest" time="0.065" classname="com.automation.tests.MultiplyServiceTest"/>
  <testcase name="multiplytest3_OneNumberAndOneSpecialCharacter_400BadRequest" time="0.03" classname="com.automation.tests.MultiplyServiceTest"/>
  <testcase name="multiplytest4_BothNegativeNumbers_200Ok" time="0.026" classname="com.automation.tests.MultiplyServiceTest"/>
  <testcase name="multiplytest5_OneNegativeInteger_200Ok" time="0.02" classname="com.automation.tests.MultiplyServiceTest"/>
  <testcase name="multiplytest6_DecimalNumbers_200Ok" time="0.044" classname="com.automation.tests.MultiplyServiceTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at com.automation.tests.MultiplyServiceTest.multiplytest6_DecimalNumbers_200Ok(MultiplyServiceTest.java:277)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- multiplytest6_DecimalNumbers_200Ok -->
  <testcase name="maxTest1_NonIntegersInSet_400BadRequest" time="0.018" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="statusCode_500_InternalServerError" time="0.016" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="unionTest1_NonIntegersInLeftSet_400BadRequest" time="0.04" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="divisionTest1_NonNumbers_400BadRequest" time="0.029" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest2_DivisorNull_400BadRequest" time="0.035" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest3_DividendNull_400BadRequest" time="0.017" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest4_DivisorSpecialCharacter_400BadRequest" time="0.041" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest5_BothNegativeNumbers_200Ok" time="0.095" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest6_DividendNegative_200Ok" time="0.032" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest6_DivisorNegative_200Ok" time="0.027" classname="com.automation.tests.DivisionServiceTest"/>
  <testcase name="divisionTest7_DividendFloat_200Ok" time="0.04" classname="com.automation.tests.DivisionServiceTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at com.automation.tests.DivisionServiceTest.divisionTest7_DividendFloat_200Ok(DivisionServiceTest.java:347)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- divisionTest7_DividendFloat_200Ok -->
  <testcase name="divisionTest8_DivisorFloat_200Ok" time="0.337" classname="com.automation.tests.DivisionServiceTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at com.automation.tests.DivisionServiceTest.divisionTest8_DivisorFloat_200Ok(DivisionServiceTest.java:388)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- divisionTest8_DivisorFloat_200Ok -->
  <testcase name="maxTest1_NonIntegersInSet_400BadRequest" time="0.033" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="maxTest2_SpecialCharactersInSet_400BadRequest" time="0.016" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="maxTest3_NegativeNumbersInSet_200Ok" time="0.022" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="maxTest4_AllNegativeNumbersInSet_200Ok" time="0.011" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="maxTest5_OneNullInSet_200Ok" time="0.013" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="minTest1_NonIntegersInSet_400_BadRequest" time="0.03" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="minTest2_SpecialCharactersInSet_400_BadRequest" time="0.032" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="minTest3_NegativeNumbersInSet_200_Ok" time="0.034" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="minTest4_AllNegativeNumbersInSet_200_Ok" time="0.137" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="minTest5_OneNullInSet_200_Ok" time="0.022" classname="com.automation.tests.MaxMinServicesTest"/>
  <testcase name="statusCode_404_NOT_FOUND" time="0.03" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="additionTest_400_Bad_Request_non_integer" time="0.025" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="statusCode_405_MethodNotAllowed" time="0.022" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="statusCode_415_UnsupportedMediaType" time="0.023" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="statusCode_401_UNAUTHORIZED_addition_path" time="0.015" classname="com.automation.tests.ErrorCodesTest"/>
  <testcase name="intersectionTest1_NonIntegersInLeftSet_400BadRequest" time="0.043" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="intersectionTest2_NonIntegersBothSets_400BadRequest" time="0.012" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="intersectionTest3_NegativeNumbersInBothSets_200Ok" time="0.019" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="intersectionTest4_NullsPresentInBothSets_200Ok" time="0.066" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest1_NonIntegersInLeftSet_400BadRequest" time="0.029" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest2_NonIntegersInRightSet_400BadRequest" time="0.043" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest3_NegativeNumbersInLeftSet_200Ok" time="0.032" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest4_NegativeNumbersInBothSets_200Ok" time="0.027" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest5_NullsInLeftSet_200Ok" time="0.02" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="unionTest6_NullsInBothSets_200Ok" time="0.046" classname="com.automation.tests.UnionIntersectServiceTest"/>
  <testcase name="additionTest1_Non_Integers_400BadRequest" time="0.032" classname="com.automation.tests.AdditionServiceTest"/>
  <testcase name="additionTest2_OneNumberAsNull_400BadRequest" time="0.018" classname="com.automation.tests.AdditionServiceTest"/>
  <testcase name="additionTest3_OneNumberAndOneSpecialCharacter_400BadRequest" time="0.041" classname="com.automation.tests.AdditionServiceTest"/>
  <testcase name="additionTest4_NegativeNumbers_200Ok" time="0.024" classname="com.automation.tests.AdditionServiceTest"/>
  <testcase name="additionTest5_OneNegativeOnePositiveNumber_200Ok" time="0.027" classname="com.automation.tests.AdditionServiceTest"/>
</testsuite> <!-- Default test -->
